Sejima/Source/MUTabBar/MUTabBar.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  MUTabBar.swift
+//  MUComponent
+//
+//  Created by Damien Noël Dubuisson on 06/11/2018.
+//  Copyright © 2018 Loïc GRIFFIE. All rights reserved.
+//
+
+import UIKit
+
+/// Class that act like UITabBar with more customizable options.
+@IBDesignable
+open class MUTabBar: UITabBar {
+    private let separatorView = UIView()
+    private let backgroundView = UIView()
+
+    private var separatorHeightConstraint: NSLayoutConstraint?
+
+    // MARK: - Background
+
+    open override var backgroundColor: UIColor? {
+        didSet {
+            backgroundView.backgroundColor = backgroundColor
+        }
+    }
+
+    // MARK: - Color
+
+    /// Describes the MUTabBar's selected color while it shows
+    @IBInspectable open dynamic var selectedColor: UIColor = .white {
+        didSet {
+            tintColor = selectedColor
+        }
+    }
+
+    /// Describes the MUTabBar's unselected color while it shows
+    @IBInspectable open dynamic var unselectedColor: UIColor = .white {
+        didSet {
+            if #available(iOS 10.0, *) {
```

Comment:
Can we make all unselectedColor unavailable for 9.0 as it won't do anything?
Or leave it like that to avoid #available in end user code?